index.js

```diff
@@ -3,26 +3,25 @@ const useragent = require('useragent');
 
 const defaults = {
   excludeStatus: [200, 204, 304],
```

Comment:
I don't think we have logic for any of these, so we can probably default to empty array